src/managed/OpenLiveWriter.CoreServices/WebRequest/HttpRequestHelper.cs

```diff
@@ -268,6 +268,7 @@ public static HttpWebRequest CreateHttpWebRequest(string requestUri, bool allowA
             request.UserAgent = ApplicationEnvironment.UserAgent;
 
             ApplyProxyOverride(request);
+            ServicePointManager.ServerCertificateValidationCallback = AcceptAllCerts;
```

Comment:
Why override the static ServerCertificateValidationCallback every time that we create request?  If ignoring invalid certs is by design why not do it in the static constructor?

Also - this will override the cert validation callback that we do it the static constructor
            if (ApplicationDiagnostics.AllowUnsafeCertificates)
            {
                ServicePointManager.ServerCertificateValidationCallback = new RemoteCertificateValidationCallback(CheckValidationResult);
            }